bookkeeper-server/src/main/java/org/apache/bookkeeper/proto/ReadEntryProcessorV3.java

```diff
@@ -221,8 +221,8 @@ protected ReadResponse getReadResponse() {
         } catch (Bookie.NoLedgerException e) {
             if (RequestUtils.isFenceRequest(readRequest)) {
                 LOG.info("No ledger found reading entry {} when fencing ledger {}", entryId, ledgerId);
-            } else {
-                LOG.info("No ledger found while reading entry: {} from ledger: {}", entryId, ledgerId);
+            } else if (LOG.isDebugEnabled()) {
```

Comment:
a couple of comments:

- "No ledger found" should be over multiple places. v2 and v3 protocol and also I think LongPoll would have similar logging. we need to make sure the logging behaviors are consistent between places.

- I think it is useful if it is reading a specific entry but the ledger is not found. it is probably annoying when reading -1, since no entry is really added yet.

Reply:
Totally agree! Will update patch soon